src/pages/StaticPage.vue

```diff
@@ -84,6 +84,10 @@ export default {
     font-size: 18px;
   }
 
+  p {
+    margin-bottom: 28px;
+  }
+
   p + p {
```

Comment:
margin between `<p>` elements is defined here with `p + p`

Reply:
Mayb try removing `p { margin-bottom}` 
- You don't want the last paragraph of the document having to big of a margin, this would add extra margin to the container.

And use `p + p { margin-top: 28px; }`

Reply:
I overlooked that the github logo is black on the mobile on live and preview version, this is now fixed.

I added the margin-bottom to all p's (including the last one) because looking at the [zeplin ](https://app.zeplin.io/project/5e7f3e9f60203a6e85a13b93/screen/5e81133e7319a9140e93f871) design the bottom spacing in the div is 69px, in this preview version it is 56px (with the added margin-bottom 28px)

I tried using ``` p + p { margin-top: 28px; }```, this is how it looks (I highlighted the selected p's with an outline)

![Screenshot_2020-05-20 COVID-19 Sledilnik (preview)](https://user-images.githubusercontent.com/52170374/82427693-ba004b80-9a89-11ea-85c3-175ba8d78de6.png)

it doesn't provide proper spacing, as it should be even between all elements, which is especially obvious in the Zbrani podatki sub-section

If the last p is a problem another solution would be: ```p:not(:last-of-type) {
    margin-bottom: 28px;
  }```

this solution provides proper spacing and doesn't disturb the container padding neither on big not mobile screens